### ISSUE TITLE
Stuck on the progress bar warning message when using global Interpretation on keras sequential model

### ISSUE THREAD

**Issue by deveshcode** [oracle/Skater#298]
I was able to use skater library functions without much problems for other flavours and models like Random forest, Gradient Booster, Decision Tree etc. to create PDPs and Feature Importances 

However, when implementing the same for Sequential Models using keras library, kernel keeps on running with just the initial warning message. 

```
from skater.core.explanations import Interpretation
from skater.model import InMemoryModel
interpreter = Interpretation(X[10:],Y[10:],list(train_df.columns)[:-1])
pyintmodel = InMemoryModel(my_model.predict, examples=X[10:], feature_names = list(train_df.columns)[:-1])
fi = interpreter.partial_dependence.plot_partial_dependence([0,1], pyintmodel)

2020-03-12 16:21:20,483 - skater.core.explanations - WARNING - Progress bars slow down runs by 10-20%. For slightly 
faster runs, do progressbar=False

```
Is the keras library / neural network models not supported in skater ? 
Also I am attaching the csv files and my jupyter notebook if that helps ( change extension 
from txt to ipynb and csv respectively)
[hourly_wages_data.txt](https://github.com/oracle/Skater/files/4327880/hourly_wages_data.txt)

[Wages_NN_Skater.txt](https://github.com/oracle/Skater/files/4327877/Wages_NN_Skater.txt)
